doc/iterations/2024.3/shapes/add_filters_on_explorer_view.adoc

```diff
@@ -0,0 +1,44 @@
+= Add filters on Explorer view
+
+== Problem
+
+In Sirius Web, the Explorer view displays tree containing data.
+These data comes from the backend where specifier have total control.
+But, once displayed in the Explorer view, end-users cannot (graphically) modify the tree.
+
+== Key Result
+
+End-users have the possibility to hide objects contained in trees in the Explorer view.
+
+== Solution
+
+Filters action should be available in the Explorer view toolbar, allowing to hide objects contained the trees in the Explorer view.
+The filters action is a menu, containing a list of filters.
+A filter menu item is a checkbox associated to a label.
+When the checkbox is enabled, the filter is enabled on all trees in the Explorer view.
+When the checkbox is disabled, the filter is disabled on all trees in the Explorer view.
+
+== Scenario
+
+End-users want to hide all elements of some type, or based on some condition.
+The filter could lead to hide an element and all its children, recursively, or not.
+Depending on the filter, an element could be hidden, but not its children.
```

Comment:
And if we go with transformations of the tree instead of filtering, I'm not even sure that `transformation1(transformation2(tree))` would be equal to `transformation2(transformation1(tree))` so the order in which those transformations would be applied would matter.

Reply:
Indeed, we will need to discuss more about that.
I definitely want the first approach.